variables.md

```diff
@@ -35,6 +35,7 @@ DUD_ADDONS | string | | Comma separated list of addons added using DUD.
 DVD |||
 ENCRYPT | boolean | false | Enables or indicates encryption of the disks. Can be combine with `FULL_LVM_ENCRYPT`, `ENCRYPT_CANCEL_EXISTING`, `ENCRYPT_ACTIVATE_EXISTING` and `UNENCRYPTED_BOOT`.
 EVERGREEN |||
+EXIT_AFTER_START_INSTALL |boolean|false| Indicates that test suite will be finished after `installation/start_install` test module. So that all the test modules after this one will not be scheduled and executed.
```

Comment:
missing some space among pipes

Reply:
Thank you. Yesterday we fought with Tidy as it passed on code with any formatting and I didn't check all the formatting things. 

But I guess this .md file is not validated by the tidy script, so it's great that you noticed this.